packages/plugins/connection-manager/language-server.ts

```diff
@@ -197,7 +198,12 @@ export default class ConnectionManagerPlugin implements ILanguageServerPlugin {
       progressBase && this.server.sendNotification(ProgressNotificationComplete, progressBase);
       e = decorateLSException(e, { conn: req.conn });
       if (e.data && e.data.notification) {
-        delete e.data.args.conn;
+
+        // To fix https://github.com/mtxr/vscode-sqltools/issues/903 we need not to delete conn from the notification.
+        // Unclear in what circumstances we *do* need to delete it, but made the minimal change to fix #903.
+        if (e.data.notification !== MissingModuleNotification) {
+          delete e.data.args.conn;
```

Comment:
@gjsjohnmurray the reason to delete credentials here is to avoid any kind of error tracking tool to log passwords. In the very beginning of the extension, I recall to be testing and I realized the extensions share context and a I believe it still remains like this.

That said, one extension (for python snippets) had a global error handling that was unhandled errors thrown by any extension, to avoid any mistakes, I decided to delete the connection information from the notification here.

It's just that. That reminded me how dangerous it can be to monitor errors even if I want to track errors/bugs on sqltools

Ever since, even for SQLTools I stopped to send events to sentry and telemetry from sqltools is fully disabled, see
![CleanShot 2022-09-11 at 02 44 55](https://user-images.githubusercontent.com/707561/189514309-c3c0b4f5-9696-4653-8a5c-72465d7c8060.png)

I'll remove that code completely since it's just weight we are carrying, but those are the reasons behind this weird delete...